### DISCUSSION TITLE
Make the "new skin imported " popup take you to/ activate the new skin upon being clicked

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#19113] [Ideas]
As stated in the title it irks me a little that clicking on the imported skin in notifications does nothing. Dunno if that's intentional but it feels a bit off

**Comment by frenzibyte** [comment]
Beatmap and score notifications present the imported item, unlike skins which cannot be "presented". Getting this to work with the existing logic for beatmaps and scores may require further thought.

**Reply by peppy** [reply]
A skin select overlay will be a thing eventually, but I don't think there's harm in making clicking the notification change the current skin to the new one.
